src/templates/all/.make/rust.mk

```diff
@@ -110,5 +111,28 @@ rust-tidy: check-cargo-registry rust-docker-pull
 	@echo "$(CYAN)Running rust file formatting fixes...$(SGR0)"
 	@$(call cargo_run,fmt,--all)
 
+rust-openapi: check-cargo-registry rust-docker-pull rust-build
+	@echo "$(CYAN)Generating openapi documentation...$(SGR0)"
+	mkdir -p $(OUTPUTS_PATH)
+	@$(call cargo_run,run,-- --openapi ./out/$(PACKAGE_NAME)-openapi.json)
+
+rust-validate-openapi: rust-openapi
+	@docker run \
+		--rm \
+		-v $(OUTPUTS_PATH):/out \
+		jeanberu/swagger-cli \
+		swagger-cli validate /out/$(PACKAGE_NAME)-openapi.json
+
+rust-grpc-api: check-cargo-registry rust-docker-pull
```

Comment:
This target runs `rust-docker-pull` but it's not using the rust docker, so this can be removed.
I also don't think the cargo registry is needed for this target, so it doesn't need to be checked either.

Reply:
https://github.com/Arrow-air/tf-github/pull/83/commits/432dd0d9ebbfaead84ddc3aabe670c26fb2478a5